common/table.js

```diff
@@ -0,0 +1,129 @@
+(function () {
+    'use strict';
+
+    angular.module('chaise.record.table', [])
+
+    .directive('recordTable', ['UriUtils', '$rootScope', '$window', '$filter', function(UriUtils, $rootScope, $window, $filter) {
+        var postLink = function(scope) {
+            var context = $rootScope.context;
+
+            scope.tableModel = {
+                tableName:          null,
+                tableDisplayName:   null,
+                columns:            [],
+                filter:             null,
+                sortedBy:           null,
+                sortOrder:          null,
+                rowset:             null,
+                keycols:            [],
+                count:              0
+            };
```

Comment:
This purely a matter of preference. This is infinitely more readable, in my mind, than having a single space after each `:`.